tests/boot-raid1.nix

```diff
@@ -9,4 +9,8 @@ makeDiskoTest {
     machine.succeed("test -b /dev/md/boot");
     machine.succeed("mountpoint /boot");
   '';
+  extraSystemConfig = {
+    # sadly systemd-boot fails to install to a raid /boot device
+    boot.loader.systemd-boot.enable = false;
```

Comment:
Yes. This requires a dedicated non-mirrored efi partition.

Reply:
but this seems like a bug in systemd-boot. since it works fine in grub. maybe something with our install script. seems like systemd-boot does some unneccessary checks for the underlying FS of the /boot partition, so maybe we can skip those